src/Components/WordTerm.js

```diff
@@ -1,25 +1,45 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
 
-export default function WordTerm({ word, def }) {
+export default function WordTerm({ word, def, links }) {
+  const linksArray = links.split('\n');
+  linksArray.pop();
   return (
-    <>
-      <div>
-        Word:
-        {word}
-      </div>
-      <div>
-        Definition:
-        {def}
-      </div>
-      <div>
-        ------------
-      </div>
-    </>
+    <Card sx={{ minWidth: 275 }}>
+      <CardContent>
+        <Typography sx={{ fontSize: 16 }} color="text.secondary" gutterBottom>
+          Term
+        </Typography>
+        <Typography variant="h5" component="div">
```

Comment:
As we add more styling, it will be preferable to replace these "variant" fields with custom css. When you have time, research Material UI MakeStyles for when we return to these features!